preview/extensions-and-macros/antora.yml

```diff
@@ -6,6 +6,7 @@ nav:
 - modules/ROOT/nav.adoc
 asciidoc:
   attributes:
+    page-release-date: 2024-03-01
```

Comment:
Consider using ISO 8601 dates to avoid problems with timezones. 

Example date in ISO 8601 "2025-01-14T10:00:00Z" -> 10:00 UTC